docs/journal/00-introducing-the-journey.md

```diff
@@ -1,118 +1,110 @@
-Now they now have a few months to roll out a system to production.
+**Ingesting data**. The most basic task of an IoT system is getting data into the back end, reliably and at scale. Considerations include:
 
-## The Journal
+- The expected data rate (how many devices; how many events per device). 
+- Physical network requirements.
+- Message protocols. (HTTP? [AMQP](https://www.amqp.org/)?) 
+- Protocol translation. In some cases, you might need to translate output from the devices into the expected ingestion format. For example, you might have legacy devices that use non-standard protocols.   
+  
+During development, it is common to use a **device simulator**, which feeds simulated data to the system. A simulator is also useful for load-testing the back end.
 
-The purpose of this journal is to record the steps that the team at Fabrikam took, document the decisions that they made, and keep a log of any issues that they encountered. The result is intended to be a story that tells of the trials and tribulations of the team as they implemented the system.
+**Consuming data**. Once the data is in the system, what are you doing with it? Here are some typical scenarios that we've identified:
 
-Note that the purpose of this journal is not necessarily to record a _realistic_ scenario, but rather focus on the issues that are _representative_ of the actual challenges that are likely to be met by most organizations faced with building an IoT system for the first time. Additionally, the specification of the system being developed by Fabrikam is not yet fixed; features might be removed or replaced and new features added. We aim to use the [backlog][] to record and drive these changes.
+- Short-term data aggregation: "In the last 5 minutes, what was the average temperature?"
+- Critical alerts: "This unit is overheating!"
+- Query the data (not in real time): "In the past week, how many times did the temperature fall outside of a given range for more than ten minutes?"
+- Long-term storage
+	- Auditing 
+	- History
+- Visualization. Business reporting and charting, dashboards, etc.
 
+**Command and control**. In some IoT scenarios, a device can receive outbound messages from the back end. This doesn't apply to every IoT solution. Our example scenario does not include command and control.   
 
-## The People
+**Backend services**. These are some things that you might build on the back end:
 
-A well-balanced team for building an IoT solution is composed of several roles. The roles that are most important for your solution may be different from the ones discussed here.
+- Dashboards.
+- System administration, to control who has access to which parts of the system. 
+- APIs. You might create API layers so that consumers don't have to deal with raw data. APIs can also help protect privacy, and handle authorization.
+- Multi-tenancy. Does your IoT solution have multiple users/customers?
```

Comment:
As an aside, @tolandmike mentioned to me that he has yet to see an IoT scenario that _wasn't_ mutli-tenant in one way or another.